Microsoft.Toolkit.Uwp.UI.Animations/Extensions/AnimationExtensions.cs

```diff
@@ -89,6 +91,56 @@ public static EasingFunctionBase GetEasingFunction(EasingType easingType)
             }
         }
 
+        /// <summary>
+        /// Generates an <see cref="CompositionEasingFunction"/> to be used with Composition animations
+        /// </summary>
+        /// <param name="easingType">The <see cref="EasingType"/> used to generate the CompositionEasingFunction</param>
+        /// <param name="compositor">The <see cref="Compositor"/></param>
+        /// <returns><see cref="CompositionEasingFunction"/></returns>
+        public static CompositionEasingFunction GetCompositionEasingFunction(EasingType easingType, Compositor compositor)
+        {
+            if (DesignTimeHelpers.IsRunningInLegacyDesignerMode)
+            {
+                return compositor.CreateLinearEasingFunction();
+            }
+
+            return GenerateCompositionEasingFunctionFromEasingType(easingType, compositor);
+        }
+
+        private static CompositionEasingFunction GenerateCompositionEasingFunctionFromEasingType(EasingType easingType, Compositor compositor)
+        {
+            if (easingType == EasingType.Default)
+            {
+                easingType = DefaultEasingType;
+            }
+
+            switch (easingType)
+            {
+                case EasingType.Linear:
+                    return compositor.CreateLinearEasingFunction();
+                case EasingType.Cubic:
+                    return compositor.CreateCubicBezierEasingFunction(new Vector2(0.215f, 0.61f), new Vector2(0.355f, 1f));
+                case EasingType.Back:
+                    return compositor.CreateCubicBezierEasingFunction(new Vector2(0.175f, 0.885f), new Vector2(0.32f, 1.275f));
+                case EasingType.Bounce:
+                    return compositor.CreateCubicBezierEasingFunction(new Vector2(0.58f, 1.93f), new Vector2(.08f, .36f));
+                case EasingType.Elastic:
+                    return compositor.CreateCubicBezierEasingFunction(new Vector2(0.37f, 2.68f), new Vector2(0f, 0.22f));
+                case EasingType.Circle:
+                    return compositor.CreateCubicBezierEasingFunction(new Vector2(0.075f, 0.82f), new Vector2(0.165f, 1f));
+                case EasingType.Quadratic:
+                    return compositor.CreateCubicBezierEasingFunction(new Vector2(0.25f, 0.46f), new Vector2(0.45f, 0.94f));
+                case EasingType.Quartic:
+                    return compositor.CreateCubicBezierEasingFunction(new Vector2(0.165f, 0.84f), new Vector2(0.44f, 1f));
+                case EasingType.Quintic:
+                    return compositor.CreateCubicBezierEasingFunction(new Vector2(0.23f, 1f), new Vector2(0.32f, 1f));
+                case EasingType.Sine:
+                    return compositor.CreateCubicBezierEasingFunction(new Vector2(0.39f, 0.575f), new Vector2(0.565f, 1f));
+                default:
+                    throw new ArgumentOutOfRangeException(nameof(easingType), easingType, null);
```

Comment:
How about NotSupportedExpection. If a new type is added and we don't support it, then we should throw NotSupportedExpection

Reply:
Changed